### PR TITLE
Updated CONTRIBUTING.md with images

### PR DESCRIPTION
**Pull request by LeeRenJie**
## Issue that this pull request solves
New images that load for CONTRIBUTING.md file

 Closes: #330 

## Proposed changes
Add images for the CONTRIBUTING.md file to guide beginners

### Brief description of what is fixed or changed
Added 7 images including:
1. PR template 
2. New branch remote directory
3. How to create branches 
4. How to compare files 
5. Where to find the PR 
6. How to find the issues 
7. Where to fork project

## Types of changes

_Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation update (Documentation content changed)
- [ ] Other (please describe): 

## Checklist

_Put an `x` in the boxes that apply_

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] My changes does not break the current system and it passes all the current test cases.

## Screenshots
new look

Please attach the screenshots of the changes made in case of change in user interface

## Other information

Any other information that is important to this pull request
All the images can be found at assets/cont-img

### CODE REVIEW

**Comment by LeeRenJie** [comment]
I am under DWOC. Happy to complete this issue. Please review my PR and provide feedbacks Thanks!!

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -109,11 +116,17 @@ A good rule of thumb for the commit message is to have a present tense verb, fol
 $ git push -u origin <branch_name>  
 ```  
 - Here is how your branch will look.  
-![br](https://github.com/kanishkaa24/Community-Website/blob/main/3.png)  
+![br](https://github.com/LeeRenJie/Community-Website/blob/main/assets/cont-img/br.png)  
   
 ## Step 6 : Pull Request  🎣
```
Comment (tejan-singh) [inline CONTRIBUTING.md:121]: You missed Step 4 and 5

**Review by tejan-singh — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (LeeRenJie) [reply CONTRIBUTING.md:121]: There were no images initially so I thought it was unnecessary. Should I add?

Comment (LeeRenJie) [reply CONTRIBUTING.md:127]: Done with step 4 & 5! Please review thanks!

**Review by tejan-singh — APPROVED** [review]

**Review by Kajol-Kumari — APPROVED** [review]
LGTM :+1:

**Comment by LeeRenJie** [comment]
Thank you for approving👍

**Pull request merged**